src/app/config.ts

```diff
@@ -27,15 +43,75 @@ const config = convict({
 			env: 'HTTP_PORT',
 		},
 	},
-	mongodb: {
-		url: {
-			doc: 'MongoDB connection URL for the main backend database',
-			default: 'mongodb://localhost:27017/readercritics',
-			env: 'MONGODB_URL',
+	db: {
+		mongo: {
+			url: {
+				doc: 'MongoDB connection URL for the main backend database',
+				format: String,
+				default: 'mongodb://localhost:27017/readercritics',
+				env: 'MONGODB_URL',
+			},
+		},
+		redis: {
+			url: {
+				doc: 'Redis URL to connect to (including auth string)',
+				format: String,
+				default: 'redis://localhost:6379',
+				env: 'REDIS_URL',
+			},
+			host: 'localhost',
+			port: 6379,
+			ttl: 260,
+		},
+	},
+	auth: {
+		bcrypt: {
+			rounds: {
+				doc: 'Number of salt rounds when hashing passwords with BCrypt',
+				default: 14,  // takes ~900ms on a Skylake E5v3 Xeon
+				env: 'AUTH_BCRYPT_ROUNDS',
+			},
+		},
+		jwt: {
+			secret: {
+				default: '1BAAF7B9ECF206FB500B1AF0DBD776E32A2B35EE1BE59A784C892008596B7DFB',
```

Comment:
The default values for auth.[jwt|session].secret should be `null` in the checked in file. This way, people will be forced to configure these values on their own and not use the defaults, which would be insecure because everyone could get these values from here and start snatching sessions.